pmpro-variable-pricing.php

```diff
@@ -423,12 +423,12 @@ function pmprovp_load_scripts() {
 	}
 
 	// Bail if PMPro is not loaded.
-	if ( ! function_exists( 'pmpro_getOption' ) ) {
+	if ( ! function_exists( 'pmpro_init' ) ) {
```

Comment:
```suggestion
	if ( ! function_exists( 'pmpro_getGateway' ) ) {
```

Reply:
For clarity, checking for a function  name that is later called in this `pmprovp_load_scripts()` function